### PR TITLE
Python2 out

### PR DESCRIPTION
**Pull request by pjhaest**
Change of quickstart.rst to smaller bbox
removed Python 2 references from the code base (?)
did not rerun the notebooks though

### CODE REVIEW

**Comment by pjhaest** [comment]
the also resolves #225

File: pydov/search/abstract.py [hunk 1]
```diff
@@ -623,7 +623,6 @@ def _search(self, location=None, query=None, return_fields=None,
                 filter_request = etree.tostring(filter_request,
                                                 encoding='unicode')
             except LookupError:
-                # Python2.7 without lxml uses 'utf-8' instead.
```
Comment (Roel) [inline pydov/search/abstract.py]: I think the whole try/except block can be removed here. This holds true for all similar changes below as well.

**Comment by johanvdw** [comment]
@pjhaest : your master branch is not in sync with the real master branch. I will rebase and split this pull request (documentation change and python2 changes)

**Comment by johanvdw** [comment]
@pjhaest : I have split this pull request in two:
https://github.com/DOV-Vlaanderen/pydov/pull/232
https://github.com/DOV-Vlaanderen/pydov/pull/233

**Pull request closed without merging**